core-services/importExport/src/main/java/org/silverpeas/core/importexport/control/ImportExport.java

```diff
@@ -499,9 +499,11 @@ private void createTopicHtmlFile(String thisExportDir, String tempDir, HtmlExpor
   }
 
   /**
+   *
    * @param userDetail
    * @param itemsToExport
-   * @return
+   * @param rootPK
+   * @return ExportPDFReport
    * @throws ImportExportException
```

Comment:
Please, leave a javadoc comment explaining what the method is expected to do and for each parameter write what they represent

Reply:
done.